sbt-metals/src/main/scala/scala/meta/metals/MetalsPlugin.scala

```diff
@@ -13,33 +13,42 @@ object MetalsPlugin extends AutoPlugin {
 
   val semanticdbVersion = BuildInfo.semanticdbVersion
 
+  private def isDottyOr210(scalaVersion: String) = {
+    scalaVersion.startsWith("2.10") || ScalaInstance.isDotty(scalaVersion)
```

Comment:
It might be better to use `supportedScalaVersions` like we do in ScalaVersions. Otherwise we might end up trying to add semanticdb for versions that do not have the support.

Reply:
Yea, good idea. I've gone ahead and re-used the `supportedScala2` for this.